### PR TITLE
Appendchild not working

### PR DESCRIPTION
**Pull request by TurnbulA**

### CODE REVIEW

File: app/js/script.js [hunk 1]
```diff
@@ -29,28 +29,28 @@ document
...
 ];
 
-//Checks if there is an active colour and removes that class
-const removeActiveColour = () => {
-  const activeColours = [...document.querySelectorAll(".isActive")];
-  activeColours.forEach(activeColour => {
-    activeColour.classList.remove("isActive");
+//Checks if there is an active color and removes that class
+const removeActivecolor = () => {
+  const activecolors = [...document.querySelectorAll(".isActive")];
```
Comment (mali891) [inline app/js/script.js]:
```
Why not just:

if (document.querySelector('.isActive')) {
  document.querySelector('.isActive').removeClass('isActive')
}
```

File: app/js/script.js [hunk 2]
```diff
@@ -60,17 +60,37 @@ const projectForm = document.querySelector("form");
...
 const createLi = () => {
-  console.log("2");
   const li = document.createElement("li");
   li.setAttribute("class", "c-icon-container");
-  console.log(li.classList);
-  projectUl.innerHTML += li;
+  const button = document.createElement("button");
+  button.setAttribute("class", "c-project--icon");
+  button.classList.add(color);
+  console.log(button.classList);
```
Comment (mali891) [inline app/js/script.js]: no

File: index.html [hunk 3]
```diff
@@ -204,13 +204,28 @@ <h1>Create a new project</h1>
...
-          <div class="c-add-project__colour--item u-icon-color--pink"></div>
-          <div class="c-add-project__colour--item u-icon-color--green"></div>
-          <div class="c-add-project__colour--item u-icon-color--blue"></div>
-          <div class="c-add-project__colour--item u-icon-color--purple"></div>
-          <div class="c-add-project__colour--item u-icon-color--orange"></div>
+        <h3>Select a color</h3>
+        <div class="c-add-project__color">
+          <div
+            class="c-add-project__color--item u-icon-color--pink"
+            data-color="u-icon-color--pink"
```
Comment (mali891) [inline index.html]: These should be in the format `u-icon--colourr-pink`

File: app/js/script.js [hunk 4]
```diff
@@ -5,24 +5,29 @@ document.querySelector(".c-burger-menu").addEventListener("click", () => {
...
-  icon.addEventListener("click", () => {
-    const projectInfo = document.querySelector(".c-project-info");
-    const bodyContainer = document.querySelector(".c-main ");
-    projectInfo.classList.toggle("c-project-info--active");
-    bodyContainer.classList.toggle("c-main--shift-left");
+const projectInfoPopup = () => {
+  const icons = [...document.querySelectorAll(".c-icon-container")];
+  icons.forEach(icon => {
+    icon.addEventListener("click", () => {
+      console.log("yo sup dawg");
```
Comment (mali891) [inline app/js/script.js]: No

Comment (TurnbulA) [reply app/js/script.js]: Yes

File: app/js/script.js [hunk 5]
```diff
@@ -5,24 +5,29 @@ document.querySelector(".c-burger-menu").addEventListener("click", () => {
...
+      projectInfo.classList.toggle("c-project-info--active");
+      bodyContainer.classList.toggle("c-main--shift-left");
+    });
   });
-});
+};
 
 //Handles opening of add project popup
+const openPopup = () => {
+  const addProject = document.querySelector(".c-add-project__popup-container");
```
Comment (mali891) [inline app/js/script.js]: `addProjectButton`

File: app/js/script.js [hunk 6]
```diff
@@ -93,9 +98,38 @@ const createLi = () => {
...
+    removeActivecolor();
+    openPopup();
+    return false;
+  } else if (!color && color.length === 0) {
+    alert("Pick a colour");
+    removeActivecolor();
+    openPopup();
+    return false;
+  }
+  // else if ((!color && color.length === 0) || inputValue.value === "") {
```
Comment (mali891) [inline app/js/script.js]: No

Comment (TurnbulA) [reply app/js/script.js]: Yes

**Pull request merged**